sklearn/neighbors/_base.py

```diff
@@ -355,7 +355,7 @@ def _check_algorithm_metric(self):
             effective_p = self.p
 
         if self.metric in ['wminkowski', 'minkowski'] and effective_p < 1:
-            raise ValueError("p must be greater than one for minkowski metric")
+            raise ValueError("p must be greater or equal to one for minkowski metric")
```

Comment:
The linter is failing because this is going over the 80 character limit:

```suggestion
            raise ValueError("p must be greater or equal to one for "
                             "minkowski metric")
```